docs/section11/Rule_11-10.md

```diff
@@ -0,0 +1,221 @@
+            - set has_fault to true: `has_fault = true`
+          - calculate the expected efficiency for VERY_SMALL draw pattern: `if draw_pattern == "VERY_SMALL": expected_UEF = 0.2674 – (0.0009 * volume)`
+          - calculate the expected efficiency for LOW draw pattern: `if draw_pattern == "LOW": expected_UEF = 0.5362 – (0.0012 * volume)`
+          - calculate the expected efficiency for MEDIUM draw pattern: `if draw_pattern == "MEDIUM": expected_UEF = 0.6002 – (0.0011 * volume)`
+          - calculate the expected efficiency for HIGH draw pattern: `if draw_pattern == "HIGH": expected_UEF = 0.6597 – (0.0009 * volume)`
+          - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL
+            - check if the actual uniform_energy_factor is >= to the expected: `if uniform_energy_factor >= expected_UEF && !has_faults:`
+              - set rule_status to "PASS": `rule_status = "PASS"`
+          - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "Uniform Energy Factor is required but not provided"`
+        - otherwise the capacity is > 105,000 btu/hr: `else:`
+          - the capacity per volume should be < 4000 btu/hr/gallon: `if capacity_per_volume >= 4000:`
+            - add the following text to the note: "For Gas Storage Water heaters, the capacity per volume should be less than 4000 btu/hr/gallon": `rule_note += "For Gas Storage Water heaters, the capacity per volume should be less than 4000 btu/hr/gallon.  "`
+            - set has_fault to true: `has_fault = true`
+          - calculate the expected_thermal_efficiency : `expected_thermal_efficiency = 0.8`
+          - calculate max_standby_loss (btu/hr): `max_standby_loss = (input_capacity/800 + 110 * sqrt(volume))`
+          - make sure that the thermal efficiency is not NULL: `if thermal_efficiency != NULL:`
+            - check whether the uniform_energy_factor is valid: `if uniform_energy_factor != NULL`
+              - check if the actual uniform_energy_factor is >= to the expected: `if shw_eq.thermal_efficiency >= expected_thermal_efficiency && standby_loss <= max_standby_loss && !has_faults:`
+                - set rule_status to "PASS": `rule_status = "PASS"`
+            - otherwise, set rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "Uniform Energy Factor is required but not provided"`
+          - otherwise set the rule status to UNDETERMINED and provide a note: `else: rule_status = "UNDETERMINED"; rule_note = "No thermal efficiency provided.  Expected thermal efficiency is: " + expected_thermal_efficiency`
+      - check if the water heater is OIL_STORAGE: `elif swh_type == "OIL_STORAGE`
```

Comment:
Oil should never be modeled in the baseline so similar to how we skip instantaenous I think we shoudl follow the same approach with Oil Storage Water Heaters.

Reply:
is oil modeled if there is no gas available on site?

Reply:
No. Oil is never modeled. Propane is modeled if no gas is available but gas=propane in terms of SWH reqs as far as i understand.